documentation/src/main/markdown/integrations/jenkinsplugin/pluginreleasenotes.md

```diff
@@ -1,5 +1,22 @@
 # Release Notes for Jenkins Plugin
 
+## **Version 9.0.0**
+**New features**
+
+* Updated to be compatible with [solution_name] 9.x.x. (Downloading and using detect9.(sh/ps1)).
```

Comment:
* This release is compatible with [solution_name] 9.x.x. (Downloading and using detect9.(sh/ps1)).